# ProjectTeleopService.srv
string teleop_command
---
bool success  # indicate success or failure